test/server/cards/02_SHD/units/GideonsLightCruiserDarkTroopersStation.spec.ts

```diff
@@ -0,0 +1,109 @@
+                context.player1.clickCard(context.gideonsLightCruiser);
+
+                // can select a villainy unit that cost 3 or less
+                expect(context.player1).toBeAbleToSelectExactly([context.scoutBikePursuer, context.phaseiiiDarkTrooper]);
+                expect(context.player1).not.toHavePassAbilityButton();
+
+                // play scout bike pursuer
+                context.player1.clickCard(context.scoutBikePursuer);
+
+                // unit should be played as free
+                expect(context.player2).toBeActivePlayer();
+                expect(context.scoutBikePursuer).toBeInZone('groundArena');
+                expect(context.player1.exhaustedResourceCount).toBe(8);
+
+                // reset
+                context.player1.moveCard(context.gideonsLightCruiser, 'hand');
+                context.player2.passAction();
+
+                // play gideon's light cruiser again, phase 3 trooper should be played automatically
+                context.player1.clickCard(context.gideonsLightCruiser);
+
+                expect(context.player2).toBeActivePlayer();
+                expect(context.phaseiiiDarkTrooper).toBeInZone('groundArena');
+                expect(context.player1.exhaustedResourceCount).toBe(16);
+            });
+
+            it('should play a villainy unit from hand or discard as we control moff gideon (as unit)', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['gideons-light-cruiser#dark-troopers-station', 'scout-bike-pursuer'],
+                        discard: ['phaseiii-dark-trooper', 'atst', 'battlefield-marine'],
+                        leader: { card: 'moff-gideon#formidable-commander', deployed: true }
+                    },
+                    player2: {
+                        discard: ['black-sun-starfighter']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.gideonsLightCruiser);
+
+                // can select a villainy unit that cost 3 or less
+                expect(context.player1).toBeAbleToSelectExactly([context.scoutBikePursuer, context.phaseiiiDarkTrooper]);
+                expect(context.player1).not.toHavePassAbilityButton();
+
+                // play scout bike pursuer
+                context.player1.clickCard(context.scoutBikePursuer);
+
+                // unit should be played as free
+                expect(context.player2).toBeActivePlayer();
+                expect(context.scoutBikePursuer).toBeInZone('groundArena');
+                expect(context.player1.exhaustedResourceCount).toBe(8);
+
+                // reset
+                context.player1.moveCard(context.gideonsLightCruiser, 'hand');
+                context.player2.passAction();
+
+                // play gideon's light cruiser again, phase 3 trooper should be played automatically
```

Comment:
For this one could we actually do it in the reverse order and confirm that if there is no target in discard and a target in hand, it lets you choose to do nothing in line with the hidden information rule. You probably would need to use `.toHavePassSingleTargetPrompt()`, can see an example of that in Alliance Dispatcher tests

Reply:
@AMMayberry1 did you reopen this comment ? I reverse order on the second test. So I have a test with pass option in hand and no pass option on discard